autoload/test/javascript/mocha.vim

```diff
@@ -4,6 +4,7 @@ endif
 
 function! test#javascript#mocha#test_file(file) abort
   return a:file =~# g:test#javascript#mocha#file_pattern
+	&& test#javascript#has_package('mocha')
```

Comment:
The reason why `intern` tests aren't working is because you need to prepend this line with `\`, otherwise Vim won't consider this part of the previous line.

```vim
   return a:file =~# g:test#javascript#mocha#file_pattern
     \ && test#javascript#has_package('mocha')
```

Also, make sure you're using spaces and not tabs 😉